.changeset/smart-meals-camp.md

```diff
@@ -0,0 +1,5 @@
+---
+"apps-rendering-api-models": minor
```

Comment:
I think this needs to be a `major` version bump if we are adding a new `required` field. That's because anyone who already uses the `FormField` struct would not be able to upgrade to this version automatically - they need to make code changes to ensure the `hidden` field has a value.